agent/structs/structs.go

```diff
@@ -2438,6 +2478,15 @@ func (r *KeyringResponses) New() interface{} {
 	return new(KeyringResponses)
 }
 
+// String converts message type int to string
+func (m MessageType) String() string {
+	s, ok := requestTypeStrings[m]
+	if ok {
+		return s
+	}
+	return "Unknown(" + strconv.Itoa(int(m)) + ")"
```

Comment:
Enterprise has some of its own requests types too. So somewhere (maybe in this function) you will need to handle calling a build tag gated function to turn the enterprise request types into strings.